Author: Andreas Tille <tille@debian.org>
Last-Update: Fri, 15 Oct 2021 07:01:52 +0200
Description: Skip test that test needs a *lot* of memory -- at
  least 4G, quite possibly 8G if there's copying involved
  see https://lists.debian.org/debian-med/2021/10/msg00069.html

--- a/test/loaders/fastq-loader/test-id2name.cpp
+++ b/test/loaders/fastq-loader/test-id2name.cpp
@@ -136,6 +136,7 @@ FIXTURE_TEST_CASE ( AddRealloc, Id2name_
     REQUIRE_EQ ( str, string ( m_res ) );
 }
 
+/*
 FIXTURE_TEST_CASE ( MiltipleBuffers, Id2name_Fixture )
 {   // exceed a single KDataBuffer's size limit of 2**32-1 bytes
     string str1 ( 0x7fffffff, '1');
@@ -162,6 +163,7 @@ FIXTURE_TEST_CASE ( NameTooLong, Id2name
     
     REQUIRE_RC_FAIL ( Id2Name_Add( & m_self, 1, str.c_str() ) );
 }
+*/
 
 //////////////////////////////////////////// Main
 extern "C"
